client/components/create-event.jsx

```diff
@@ -30,28 +34,55 @@ export default function Event() {
                 message: 'Minimium event name is 4'
               },
               maxLength: {
-                value: 10,
-                message: 'Maximum event name is 10'
+                value: 20,
+                message: 'Maximum event name is 20'
               }
             })} />
           </div>
           <div>
-            <p>{errors.name?.message}</p>
+            <p>{errors?.name?.message}</p>
+          </div>
+        </label>
+      </div>
+      <div>
+        <label> <span className='text-white'>Start Date and Time</span>
+          <div>
+            <Controller
+            name="startDate"
+            control={control}
+            required render={({ field }) => (
+              <Datetime
+              {...field}
+              />
+            )}
+             />
+          </div>
+        </label>
+      </div>
+      <div>
+        <label><span className='text-white'>End Date and Time</span>
+          <div>
+            <Controller
+              name="endDate"
+              control={control}
+              required render={({ field }) => (
```

Comment:
I'm curious, if you just put `required` here, does RHF refuse to submit the form if it hasn't been specified? If so, that's awesome.